schematic/manifest/generator.py

```diff
@@ -1589,7 +1588,7 @@ def create_single_manifest(
             use_annotations (bool, optional): whether to use annotations. Defaults to False.
 
```

Comment:
The docstring doesn't match the method arguments. Could you re-write to match?

Reply:
That's a good point. I have re-generated the docstring!